pkg/util/netifcmd/ifconfig.go

```diff
@@ -0,0 +1,109 @@
+package netifcmd
+
+import (
+	"net"
+	"os"
+
+	"k8s.io/klog/v2"
+	utilexec "k8s.io/utils/exec"
+)
+
+const (
+	cmdIfconfig string = "ifconfig"
+	cmdSysctl   string = "sysctl"
+)
+
+// runner implements Interface in terms of exec("ifconfig").
+//
+// This implementation is darwin specific, but may work on other platforms.
+type ifConfigDarwinRunner struct {
+	exec          utilexec.Interface
+	checkIPExists func(string, net.IP) (bool, error)
+}
+
+// Ensure ifConfigDarwinRunner implements Interface
+var _ Interface = &ifConfigDarwinRunner{}
+
+// Returns a new implementation of Interface to execute the Darwin interface management tool: ifconfig.
+// Uses the interface defined in INTERFACE_TO_ADD_SERVICE_IP environment variable, if this value is empty uses the interface en0.
+func NewIfconfigDarwin(exec utilexec.Interface) (Interface, error) {
+	var ipForwardingEnableArgs = []string{"-w", "net.inet.ip.forwarding=1"}
+	output, err := exec.Command(cmdSysctl, ipForwardingEnableArgs...).CombinedOutput()
```

Comment:
There's `pkg/util/sysctl/sysctl.go`, so we don't have to rely on an external command.

Reply:
Unfortunately this is a Linux specific module. It writes to /proc/sys to avoid calling sysctl, but there’s no such thing as /proc/sys under MacOS :-(